react/src/components/ModelTryContentButton.tsx

```diff
@@ -236,33 +230,149 @@ const ModelTryContentButton: React.FC<ModelTryContentButtonProps> = ({
     return {
       serviceName: `${modelName}-${generateRandomString(4)}`,
       replicas: 1,
-      environments: modelCardMetadata?.environments || {
+      environments: {
         environment: '',
         version: '',
         image: null,
       },
-      runtimeVariant: modelCardMetadata?.runtimeVariant || 'custom',
+      runtimeVariant: 'vllm',
       cluster_size: 1,
       cluster_mode: 'single-node',
       openToPublic: true,
       resourceGroup: currentResourceGroupByProject as string,
-      resource: modelCardMetadata?.resource || {
+      resource: {
         cpu: 4,
         mem: '32g',
         accelerator: 10,
         acceleratorType: 'cuda.shares',
         shmem: '1g',
       },
-      vFolderID: vfolderID, // TODO: add cloned folder result
+      vFolderID: vfolderID,
       modelMountDestination: '/models',
       modelDefinitionPath: '',
       mount_id_map: {},
-      envvars: modelCardMetadata?.envvars || [],
+      envvars: [],
       enabledAutomaticShmem: false,
     };
   };
 
-  const cloneOrCreateModelService = (runtimeVariant: string) => {
+  // Check if folder has required definition files
+  const checkFolderDefinitionFiles = async (vfolderId: string) => {
```

Comment:
How about running `checkFolderDefinitionFiles` during render so that it can be suspended using a fallback?

If `service-definition.toml` is not existed, we can disable or hide this button.